src/topology/instances/ennreal.lean

```diff
@@ -257,6 +257,45 @@ begin
   { exact hβ, },
 end
 
+lemma tendsto_sub {a b : ℝ≥0∞} (h : a ≠ ∞ ∨ b ≠ ∞) :
+  tendsto (λ p : ℝ≥0∞ × ℝ≥0∞, p.1 - p.2) (𝓝 (a, b)) (𝓝 (a - b)) :=
```

Comment:
Is this a job for `continuous_at`?
```suggestion
  continuous_at (λ p : ℝ≥0∞ × ℝ≥0∞, p.1 - p.2) (a, b) :=
```

Reply:
All the other lemmas for other operations are formulated with tendsto, so I wouldn't change just for this one.